### PR TITLE
luci-app-ipsec-vpnserver-manyusers: code alignment

### PR DESCRIPTION
**Pull request by Beginner-Go**

### CODE REVIEW

File: applications/luci-app-ipsec-vpnserver-manyusers/root/etc/init.d/ipsecvpn [hunk 1]
```diff
@@ -65,4 +65,4 @@ EOF
 stop() {
 	ps -w | grep "/usr/lib/ipsec" | grep -v "grep" | awk '{print $1}' | xargs kill -9 >/dev/null 2>&1
 	fw3 -q reload 2>&1
-}
\ No newline at end of file
+}
```
Comment (1715173329) [inline applications/luci-app-ipsec-vpnserver-manyusers/root/etc/init.d/ipsecvpn:68]: Again don't rename uci-defaults

File: applications/luci-app-ipsec-vpnserver-manyusers/luasrc/model/cbi/ipsec-server/settings.lua [hunk 2]
```diff
@@ -1,31 +1,33 @@
...
-local m, s, o
+local mp, s, o
+
 mp = Map("ipsec", translate("IPSec VPN Server"))
-mp.description = translate(
-                     "IPSec VPN connectivity using the native built-in VPN Client on iOS or Andriod (IKEv1 with PSK and Xauth)")
-mp.template = "ipsec-server/index"
+mp.description = translate("IPSec VPN connectivity using the native built-in VPN Client on iOS or Andriod (IKEv1 with PSK and Xauth)")
+
+mp.template = "ipsec-server/ipsec-server_index"
```
Comment (1715173329) [inline applications/luci-app-ipsec-vpnserver-manyusers/luasrc/model/cbi/ipsec-server/settings.lua:9]: Why?

File: applications/luci-app-ipsec-vpnserver-manyusers/luasrc/model/cbi/ipsec-server/settings.lua [hunk 3]
```diff
@@ -1,31 +1,33 @@
...
 
 s = mp:section(TypedSection, "service")
 s.anonymous = true
+
 o = s:option(DummyValue, "ipsec-server_status", translate("Current Condition"))
-o.template = "ipsec-server/status"
+o.template = "ipsec-server/ipsec-server_status"
```
Comment (1715173329) [inline applications/luci-app-ipsec-vpnserver-manyusers/luasrc/model/cbi/ipsec-server/settings.lua:15]: Why?

File: applications/luci-app-ipsec-vpnserver-manyusers/luasrc/model/cbi/ipsec-server/settings.lua [hunk 4]
```diff
@@ -1,31 +1,33 @@
...
+
 enabled = s:option(Flag, "enabled", translate("Enable"))
```
Comment (1715173329) [inline applications/luci-app-ipsec-vpnserver-manyusers/luasrc/model/cbi/ipsec-server/settings.lua:17]: All of these custom variables `enabled` `clientip` etc. are pointless. Replace them with `o`.

File: applications/luci-app-ipsec-vpnserver-manyusers/luasrc/model/cbi/ipsec-server/settings.lua [hunk 5]
```diff
@@ -1,31 +1,33 @@
 local s = require "luci.sys"
```
Comment (1715173329) [inline applications/luci-app-ipsec-vpnserver-manyusers/luasrc/model/cbi/ipsec-server/settings.lua:1]: This definition has been overrided, where it got called?

**Review by 1715173329 — CHANGES_REQUESTED** [review]

**Comment by Beginner-Go** [comment]
changes completed

File: applications/luci-app-ipsec-vpnserver-manyusers/root/etc/init.d/ipsecvpn (hunk 1 above)
Comment (Beginner-Go) [reply applications/luci-app-ipsec-vpnserver-manyusers/root/etc/init.d/ipsecvpn:68]: Does not affect operation

File: applications/luci-app-ipsec-vpnserver-manyusers/luasrc/model/cbi/ipsec-server/settings.lua (hunk 2 above)
Comment (Beginner-Go) [reply applications/luci-app-ipsec-vpnserver-manyusers/luasrc/model/cbi/ipsec-server/settings.lua]: Does not affect operation

**Pull request closed without merging**